core/lib/dal/migrations/20240320173835_consistency-checker-cursor.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE consistency_checker_info
```

Comment:
I think it could make sense to have a more general singleton table holding cursors for L1 batch (and miniblock?) processing; it could become useful if we split L1 batch processing as planned (e.g., separate initial writes / protective reads processing).

Reply:
It could very well make sense, but I'd be for switching to such table only when we introduce new batch processing component as design of such table should be really well thought